store/src/sql/entities/query.rs

```diff
@@ -34,8 +36,21 @@ impl EventQuery {
             ORDER BY eb.idx;"#
     }
 
-    /// Requires binding 4 parameters. Finds the `EventBlockRaw` values needed to rebuild the event
-    pub fn value_blocks_many() -> &'static str {
+    /// Requires binding 1 parameter. Finds the `EventValueRaw` values needed to rebuild the event
+    /// Looks up the event by the root CID of the event.
+    /// TODO(stbrody): No need to return order_key?
```

Comment:
yeah, it looks like you're right. you only use this to get a BlockRow so it should order_key shouldn't be necessary.